data/journal_check/bug_refs.json

```diff
@@ -0,0 +1,164 @@
+{
+    "bsc_1022527_FEATURE": {
+        "description": ".*wickedd.*ni_process_reap.*blocking waitpid.*",
+        "products": {
+            "sle-micro": ["5.0", "5.1"],
+            "opensuse": ["15.2", "15.3", "Tumbleweed"],
+            "microos": "Tumbleweed"
+          }
+    },
+    "bsc_1126272": {
+        "description": "Failed unmounting \/\\S+\\.|-- Reboot --|pam_systemd.*Failed to release session",
+        "products": {
+            "sle-micro": ["5.0", "5.1"],
+            "microos": "Tumbleweed"
+          }
+    },
+    "bsc_1022525_FEATURE": {
+        "description": ".*rpcbind.*cannot(.*open file.*rpcbind.xdr.*|.*open file.*portmap.xdr.*|.*save any registration.*)",
+        "products": {
+            "sle-micro": ["5.0", "5.1"],
+            "opensuse": ["15.2", "15.3", "Tumbleweed"]
+          }
+    },
+    "bsc_1062349": {
+        "description": ".*vbd.*xenbus_dev_probe on device.*",
+        "products": {
+            "sle-micro": ["5.0", "5.1"],
+            "opensuse": ["15.2", "15.3", "Tumbleweed"]
+          }
+    },
+    "bsc_1022524_FEATURE": {
```

Comment:
This is actually https://bugzilla.suse.com/show_bug.cgi?id=1177462 and also shouldn't be ignored

Reply:
Yes, please keep this one although it's close. This is still being reported in [SLE Micro](https://openqa.suse.de/tests/6251886#step/journal_check/21)

Reply:
Ok, adding this bsc#1184970